ipn/ipnlocal/local.go

```diff
@@ -3220,6 +3220,10 @@ func (b *LocalBackend) TailscaleVarRoot() string {
 	switch runtime.GOOS {
 	case "ios", "android", "darwin":
 		return paths.AppSharedDir.Load()
+	case "linux":
+		if distro.Get() == distro.Gokrazy {
+			return "/perm/tailscaled"
```

Comment:
Why is hard-coding this path necessary here? Since https://github.com/gokrazy/gokrazy/commit/e2c5a62f677847d491591be77950f1dba084eda5, we use `/perm/home/tailscaled` for new installations. I wonder if we should update the path here to match the new default path? Or avoid it entirely somehow?